requirements.txt

```diff
@@ -19,8 +19,9 @@ google-cloud-bigquery==3.5.0
 google-cloud-billing==1.9.1
 ibm_platform_services==0.27.0
 myst-parser==1.0.0
+numpy==1.26.4
```

Comment:
@athiruma, failed use 1.24.4 instead

Reply:
pls add comment with opensearch 1.2.0 for numpy/pandas/elasticsearch

Reply:
done